File: A2Z_Sheet/Step 1 - Learn the basics/Lec 2: Build—up Logical Thinking/pattern7.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
      for(int i=0;i<5;i++){
        for(int k=5;k>i+1;k--){
          System.out.print(" ");
        }
        for(int j=0;j<2*i+1;j++){
          System.out.print("*");
        }
        System.out.println();
      }
  }
}

/*
Output:

    *
   ***
  *****
 *******
*********

*/

File: A2Z_Sheet/Step 1 - Learn the basics/Lec 2: Build—up Logical Thinking/pattern7.cpp
#include <iostream>
using namespace std;

int main() 
{
    for(int i=0;i<5;i++){
      for(int k=5;k>i+1;k--){
          cout<<" ";
        }
      for(int j=0;j<2*i+1;j++){
        cout<<"*";
      }
      cout<<endl;
    }
}

/*
Output:

    *
   ***
  *****
 *******
*********

*/
